pkg/standalone/standalone.go

```diff
@@ -51,6 +51,13 @@ const (
 	redisDockerImageName       = "redislabs/rejson"
 	zipkinDockerImageName      = "openzipkin/zipkin"
 
+	githubContainerRegistryName = "ghcr"
+	ghcrURL                     = "ghcr.io/dapr"
+	daprGhcrImageName           = "dapr"
+	dockerContainerRegistryName = "dockerhub"
+	redisGhcrImageName          = "3rdparty/redis"
```

Comment:
3rdparty/rejson
See PR https://github.com/dapr/dapr/pull/4411. Once this is merged it should be changed.

Reply:
yes, agree